buildenv/jenkins/openjdk_tests

```diff
@@ -113,7 +113,11 @@ if (PLATFORM_MAP.containsKey(params.PLATFORM)) {
     LABEL = params.LABEL ? params.LABEL : PLATFORM_MAP[params.PLATFORM]["LABEL"]
 
     if (params.DOCKER_REQUIRED) {
-        LABEL="${LABEL}&&sw.tool.docker"
+        LABEL += "&&sw.tool.docker"
+    }
+
+    if (params.LABEL_ADDITION) {
+        LABEL += "&&${params.LABEL_ADDITION}"
```

Comment:
Do you want to keep it to only additive &&, or leave it to users to supply a string that includes the "&&" or "||"?  I am fine with this, as it ensures no one forgets to add this portion of the string, just wanted to check that it covers all the cases needed.

Reply:
Good question. Originally when I proposed using existing LABEL param I thought it would be up to the user to explicitly specify. We could leave it open and prefix it with `&&` if either `&&` or `||` is not present.